### DISCUSSION TITLE
Error fetching notifications in notifications tab

### DISCUSSION THREAD

**Discussion by yagueto** [community/community#12663] [Mobile]
When I open the notifications tab in the mobile GH app, it tries to fetch the info but crashes with the error code "9BB6:13D70:2A7E18:31EA19:622A46DD" and "D036:F47F:650B9BF:6726C2B:622A49DD".
![Screenshot_20220310-194349_GitHub](https://user-images.githubusercontent.com/48566793/157734755-735a2035-25d9-4da5-9b8e-97ecfdba76cf.jpg)
![Screenshot_20220310-195633_GitHub](https://user-images.githubusercontent.com/48566793/157735150-6f7dc923-1c3f-4438-b431-413543a6dcd7.jpg)

This only happens with the "inbox" filter, the "done" works perfectly

**Comment by Iron-Ham** [comment]
Thanks for reporting this, @yagueto! 

I'll make an internal issue to track this. Could I get some more information from you? Could you let me know the version of the app you're using, and some details about your phone: OS-Version as well as phone make/model? 


Thanks!

**Reply by yagueto** [reply]
I'm using version 1.55.0 (643) of the app on a Samsung S10 with Android 12/One UI 4.0.

I've also tried accessing the notifications from my PC, but it seems like there is something wrong here also...
![imagen](https://user-images.githubusercontent.com/48566793/157754259-36c9e052-d1c5-4794-8db4-3866fd0cca2e.png)
